guidelines/releases-guidelines/02-kyma-release-process.md

```diff
@@ -33,77 +33,15 @@ To prepare a release:
 
 ### Define new release jobs
 
-Define release jobs on the `master` branch in the `test-infra` repository. To ensure every job name is unique, prefix it with `pre-rel{versionNumber}`. Remember to provide the version number without any periods. For example, to find all jobs for the 1.4 release, look for job names with the `pre-rel14` prefix. To learn how to define a release job for a component, read the following [document](https://github.com/kyma-project/test-infra/blob/master/docs/prow/release-jobs.md).
-
-1. Go to the `test-infra` repository.
-2. Define release jobs in the `prow/jobs/test-infra` directory in the `watch-pods.yaml` file.
-3. Define release jobs in the `prow/jobs/kyma` directory in the following files:
-
- * every `.yaml` in `components`
- * every `.yaml` in `tests`
- * every `.yaml` in `tools/` **except for** the `tools/failery/failery.yaml` file
- * `kyma-integration.yaml`
- * `kyma-artifacts.yaml`
- * `kyma-installer.yaml`
- * `kyma-release-candidate.yaml`
- * `kyma-github-release.yaml`
-
-   > **NOTE:** Remember to modify the `presets` array. This is an example of the 1.4 release `preset`:
-
-   ```yaml
-   - labels:
-       preset-target-commit-1.4: "true"
-       env:
-       - name: RELEASE_TARGET_COMMIT 
-           value: release-1.4
-   ```
-
-   > **NOTE:** Remember to update the `preset-target-commit-1.4` label for the `post-rel14-kyma-github-release` job. Follow the same procedure for each released version. For example, see release 1.4 configuration:
-
-   ```yaml
-   postsubmits:
-     kyma-project/kyma:
-     - name: post-rel14-kyma-github-release
-       branches:
-       - release-1.4
-       <<: *job_template
-       extra_refs:
-       - <<: *test_infra_ref
-         base_ref: release-1.4
-       labels:
-         <<: *job_labels_template
-         preset-build-release: "true"
-         preset-target-commit-1.4: "true"
-   ```
-
-4. Ensure that tests for the release jobs exist. Release tests usually iterate through all release versions and run tests for them. See the `TestBucReleases` test defined in `development/tools/jobs/kyma/service_binding_usage_controller_test.go` as a reference.
-
-5. Update the `GetAllKymaReleaseBranches()` function defined in the `development/tools/jobs/tester/tester.go` file under the `test-infra` repository.
-
-6. Define branch protection rules for the release branch in the `prow/config.yaml` file.
-   For example, see the release-1.4 definition:
-
-   ```yaml
-   release-1.4:
-     protect: true
-     required_status_checks:
-       contexts:
-         - pre-rel14-kyma-integration
-         - pre-rel14-kyma-gke-integration
-         - pre-rel14-kyma-gke-upgrade
-         - pre-rel14-kyma-gke-central-connector
-         - pre-rel14-kyma-artifacts
-         - pre-rel14-kyma-installer
-         - pre-rel14-kyma-gke-minio-gateway
-   ```
-
-### Remove old release jobs
```

Comment:
You can remove this step from the preparation list above.

Reply:
Done